avalon/photoshop/lib.py

```diff
@@ -0,0 +1,255 @@
+import json
+import contextlib
+import functools
+
+from win32com.client import Dispatch
+
+from . import com_objects
+from ..tools import html_server
+
+# Convinience variable that could later query whether the platform is Windows
+# or Mac.
+# This needs to be a partial function we can later call because when calling
+# Dispatch directly from a different thread will result first a
+# "CoInitialize has not been called.", which can be fixed with
+# pythoncom.CoInitialize() but secondly "The application called an interface
+# that was marshalled for a different thread".
+app = functools.partial(Dispatch, "Photoshop.Application")
```

Comment:
I seemed to have forgotten to actually submit this comment. Would this function also work if it's written as:
```python
def app():
    return Dispatch("Photoshop.Application")
```

In essence that should behave very similar to the `functools.partial` equivalent that you're using.
That would also make it much clearer that it's solely a convenience function, correct?